common:
    lid_topic:  "/velodyne_points"
    imu_topic:  "/imu_raw"
    lidar_type: 3                
    scan_line: 16
    blind: 1
    det_range: 100.0
    fov_degree: 180
 
mapping:
    filter_size_map: 0.5
    acc_cov: 0.1
    gyr_cov: 0.1
    b_acc_cov: 0.0001
    b_gyr_cov: 0.0001
    lid_point_cov: 0.001
    extrinsic_est_en:  false      # true: enable the online estimation of IMU-LiDAR extrinsic
    extrinsic_T: [ 0, 0, 0.28]
    extrinsic_R: [ 1, 0, 0, 
                   0, 1, 0, 
                   0, 0, 1]



publish:
    path_pub_en: true
    cur_pcl_pub_en: true      
    map_pub_en: true        
    dense_map_pub_en: false    

    path_topic: "/path"
    cur_pcl_topic: "/cur_pcl"
    map_topic:  "/map"
    dense_map_topic: "/dense_map"

save:
    map_save_type: 1        # 0 not save, 1 ikdtree map, 2 dense map;
    map_save_path: "/home/ww/slam/lslam/ww_lslam/catkin_ws/map/"